libcloudforensics/scripts/utils.py

```diff
@@ -38,14 +38,13 @@ def ReadStartupScript(filename: str = '') -> str:
   """
 
   try:
-    script_path = None
```

Comment:
This one needs to stay, because if `filename` is not `''` then the second `if` block is gonna throw an error as `script_path` isn't defined